### PR TITLE
Realtime path channel

### PR DESCRIPTION
**Pull request by dlippy**

### CODE REVIEW

File: formant_ros2_adapter/scripts/main.py [hunk 1]
```diff
@@ -415,6 +419,16 @@ def message_callback(self, topic, base_message):
                     except Exception as e:
                         print("Error ingesting " + stream + ": " + str(e))
                         return
+                elif type(message) == BotPath:
+                    try:
+                        data = message_to_json(message)
+                        self.fclient.send_on_custom_data_channel("path_info", json.dumps(data).encode("utf-8"))
+                        self.fclient.post_json(stream, message_to_json(message))
+                    except grpc.RpcError as e:
+                        return
```
Comment (ahelten) [inline formant_ros2_adapter/scripts/main.py:428]: Any reason not to print an error here?

**Review by ahelten — COMMENTED** [review]
Had one comment, otherwise looks good.

File: formant_ros2_adapter/scripts/main.py (hunk 1 above)
Comment (dlippy) [reply formant_ros2_adapter/scripts/main.py:428]: @ahelten No - we were just keeping it consistent with what formant had done and they never print an error there in the other statements.

Comment (dlippy) [reply formant_ros2_adapter/scripts/main.py:428]: I'll go ahead and merge and we can maybe look later at whether we should add additional error logging to the module.

**Pull request merged**